OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -190,26 +188,27 @@ public void ConvertIniMap(string iniFile)
 			{
 				if (kv.First <= 7)
 				{
-					var a = new ActorReference("mpspawn")
-					{
-						new LocationInit((CPos)kv.Second),
-						new OwnerInit("Neutral")
-					};
-					map.Actors.Value.Add("Actor" + map.Actors.Value.Count.ToString(), a);
+					var ar = new ActorReference("mpspawn")
+						{
```

Comment:
Style nit: Your indentation of the object initializers for various ActorReferences is inconsistent in this file.